synth.py

```diff
@@ -35,3 +35,23 @@
   )
 
 java.common_templates()
+
+# TODO: Remove all below s.replace() logic when upstream correction is made in gapic
```

Comment:
Please link to the upstream issue on gapic-generator

Reply:
Done.